router.ts

```diff
@@ -13,18 +20,53 @@ export class Router {
         app.use(bodyParser.json());
 
         app.get('/GetShipPositions', (req, res) => {
+            console.log(req.url);
             let positions = myBot.getShipPositions();
             res.send(positions);
         });
 
         app.post('/SelectTarget', (req, res) => {
+            console.log(req.url);
             let target = myBot.selectTarget(req.body);
+            console.log(target);
             res.send(target);
         });
 
         app.listen(app.get('port'), () => {
             console.log("Node app is running at localhost:" + app.get('port'));
         });
+        /*let test = new ShipPossibilities(4);
+
+        test.showPossibilities();
+
+        let testBlock: ShipPlacement = new ShipPlacement(Direction.Horizontal,5,new Position(3,3));
+
+        test.removePosibilities(testBlock);
+
+        test.showPossibilities();*/
+        
+        // let testGen: ConfigurationGenerator = new ConfigurationGenerator();
+
+        // let shots: Array<Shot> = [{Position: new Position(4,4), WasHit: false}];
+
+        // testGen.generateConfigurations(10000,[5,4,3,3,2],shots);
+        // console.log(testGen.boardCounter);

```

Comment:
I don't in general like commented out code, but I can see the value here.

(Though, in theory, you could set up the different tests as `if`s, each turned on by a `const testGeneric: boolean = true/false` variable somewhere)